processor/worker.go

```diff
@@ -58,38 +52,18 @@ type worker struct {
 
 // start starts the various worker goroutines
 func (w *worker) start() {
-	// ping loop
+	if !w.handle.config().enablePipelining {
```

Comment:
We should get rid of the `enablePipelining` config probably in future as we stopped using it